src/storage/src/hummock/event_handler/refiller.rs

```diff
@@ -382,14 +384,15 @@ impl CacheRefillTask {
             let mut admits = 0;
 
             for block_index in block_index_start..block_index_end {
-                let (range, uncompressed_capacity) = sst.calculate_block_info(block_index);
+                let (range, _uncompressed_capacity) = sst.calculate_block_info(block_index);
                 let key = SstableBlockIndex {
                     sst_id: object_id,
                     block_idx: block_index as u64,
                 };
+                // see `CachedBlock::serialized_len()`
                 let mut writer = sstable_store
                     .data_file_cache()
-                    .writer(key, key.serialized_len() + uncompressed_capacity);
+                    .writer(key, key.serialized_len() + 1 + 8 + range.size().unwrap());
```

Comment:
Could you add some comment to explain the magic number?

Reply:
I've left `// see `CachedBlock::serialized_len()` for others to refer to.